docs/en/sql-reference/functions/date-time-functions.md

```diff
@@ -180,6 +180,7 @@ Result:
 ## toYear
 
 Converts a date or date with time to a UInt16 number containing the year number (AD).
+Also accepts zero arguments and returns the current year at one of the moments of query analysis. The same as ‘toYear(now())’ or ‘toYear(today())’.
```

Comment:
From a usage POV, a zero-argument form of `toYear`/`YEAR` is nice, but the user will wonder why a similar syntax is not available for `toQuarter()`/`toMonth()`/`toDayOfYear()`/`toDayOfMonth()`/`toDayOfWeek()`/`toHour()`/`toMinute()`/`toSecond()`. It will be perceived as inconsistency. Would it be possible to implement the same also for the listed functions?

Reply:
maybe I should create the new function with name thisYear()? But it looks strange.

Reply:
@2063436123 Can't we just add this behavior to other functions?

Reply:
@evillique yes, I'm planning to realize a new function thisYear().

Reply:
Why is a separate function needed? The current solution is great and we only need to add behavior like `quarter() = quarter(now)`, ` month() = month(now())`, etc

Reply:
Sorry, I misunderstood your last comment, you are right, we should add this new syntax, of course, it will add a lot of new usages of functions.

Reply:
Sorry, I misunderstood your last comment, you are right, we should add this new syntax, of course, it will add a lot of new usages of functions.